packages/react-select/src/Select.js

```diff
@@ -415,9 +418,21 @@ export default class Select extends Component<Props, State> {
       const menuOptions = nextProps.menuIsOpen
         ? this.buildMenuOptions(nextProps, selectValue)
         : { render: [], focusable: [] };
-      const focusedValue = this.getNextFocusedValue(selectValue);
-      const focusedOption = this.getNextFocusedOption(menuOptions.focusable);
-      this.setState({ menuOptions, selectValue, focusedOption, focusedValue });
+
+      let focusedValue = null;
+      let focusedOption = null;
+
+      if (nextProps.focusDefaultOption) {
+        focusedValue = this.getNextFocusedValue(selectValue);
+        focusedOption = this.getNextFocusedOption(menuOptions.focusable);
+      }
```

Comment:
I'm not sure that this prop should affect `focusedValue`, I think it should only affect `focusedOption`. `focusedValue` refers to the focused value in a multi-select (if you hover over a selected value or expand the menu of a multi-select and press the left arrow) and his little relation to the `focusedOption`. By default there is no `focusedValue` (see [implementation](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.js#L769-L788)) unless you perform one of the above actions.